_posts/2024-01-29-strimzicon2024-announcement.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title:  "Welcome to the StrimziCon 2024!"
+date: 2024-01-26
+author: paolo_patierno
+---
+
+We are very pleased to announce the first StrimziCon coming this year!
+This is going to be the first conference focused on our beloved Strimzi and Apache Kafka projects and it will be "virtual".
+It is organized and driven by the project maintainers in collaboration with the [Cloud Native Computing Foundation](https://www.cncf.io/).
+
+<!--more-->
+
+![StrimziCon 2024 Banner](/assets/images/posts/2024-01-29-strimzicon2024-banner.png)
+
+StrimziCon is an event for developers, DevOps engineers, and solution architects who want to learn about Strimzi and event streaming.
+This event will bring together the Strimzi community, and you will learn from experts and users about use cases, best practices, and their experience using Strimzi in their organizations.
+If you are working with cloud-native technologies, and you are looking for how to deal with data in motion in a cloud-based environment, don’t miss the opportunity.
+
+It will take place on **Wednesday, May 22nd** (afternoon CEST) and will feature two parallel tracks on the following themes:
+
+* Features, How Strimzi Works.
+* Operations, Using Strimzi with other projects.
```

Comment:
```suggestion
* Track 1: How Strimzi works (features)
* Track 2: Using Strimzi with other projects (operations)
```

Reply:
At the beginning I had "Track X" as well but then I thought ... "what's the point to write Track X if the sentence before mentions parallel tracks? It's obvious these themes are related to track X and Y", or?

Reply:
Just to be clearer when glancing at the info and to structure it a bit. again, just a suggestion